RecoHGCal/TICL/plugins/PatternRecognitionbyCA.h

```diff
@@ -6,13 +6,16 @@
 #include <memory>  // unique_ptr
 #include "RecoHGCal/TICL/interface/PatternRecognitionAlgoBase.h"
 #include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+namespace tf = tensorflow;
```

Comment:
this is pretty close to "using namespace" in a header file, which is not allowed.

Reply:
Ok. I will remove it also in the .cc files for consistency.